charts/falco/templates/pod-template.tpl

```diff
@@ -16,6 +16,7 @@ metadata:
       {{- toYaml . | nindent 4 }}
     {{- end }}
 spec:
+  serviceAccountName: {{ include "falco.serviceAccountName" . }}
```

Comment:
if we dont set that it will get the default one?

Reply:
Correct.